packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -8099,6 +8099,309 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/certifiedAttributes/groups/{groupIndex}/attributes/{attributeId}:
+    parameters:
+      - name: eserviceId
+        in: path
+        description: The E-Service ID of the Descriptor
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: descriptorId
+        in: path
+        description: The ID of the Descriptor
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: groupIndex
+        in: path
+        description: The position index of the Attribute Group
```

Align with the description of groupIndex already used in other schemas (or modify also the groupIndex description in the schemas below to align with this) - same for the three calls below

```suggestion
        description: The index of the group to which the Attribute belongs
```